knowledge-base/set-culture-of-date-picker-in-angular-viewer.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Change the Culture of Kendo DatePicker Widget in the Parameters Area of the Angular Report Viewer
+description: How to Set Culture for Kendo Widgets used in Angular Report Viewer
+type: how-to
+page_title: Localize the Angular Viewer Kendo Widgets
+slug: set-culture-of-date-picker-in-angular-viewer
+position: 
+tags: 
+ticketid: 1459450
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The article describes how to change the culture of the Kendo widgets used to build the Angular Report Viewer.
```

Comment:
I`m not sure here for one thing. The Description targets Kendo widgets, as a whale, and after that in Solution we are talking about for the specific widget. May be it would be better to be one or another. I`m more for the general one, because the solution actually set the culture for the whale kendo, not specific widget. If you agree, then we should parafraza the first sentence.